### ISSUE TITLE
Does not ungrey

### ISSUE THREAD

**Issue by nagim** [BlaXpirit/gitbook-plugin-ungrey#1]
Hi,

I included the plugin in the book.json, but it does not ungrey the chapter without file.
Here is my SUMMARY.md:

``` markdown
# Summary

* [Introduction](README.md)
* [Ascii](ascii.adoc)
   * [rST](sdk.rst)
   * [Chapter 1](chapter1.md)
   * [Glossary](GLOSSARY.md)
* New section
   * [Chapter](chapter1.md)
* [Next section](chapter1.md)
```

Anything after **New section** is not visible.

Could you please help?

Thanks!

**Comment by oprypin** [comment]
This information is not enough.

Unless maybe Gitbook changed significantly which entirely broke this plugin. I currently don't have time to look into this.

**Comment by nagim** [comment]
Thanks for the reply! I would be happy to provide more information, if I could, but there's really no errors, etc. If you have ideas what logs would be useful, I could send them. I have seen your plugin working elsewhere and my syntax, conifg is the same, but cannot figure out the problem.
I will test further.

**Comment by BlaXpirit** [comment]
I rebuilt [this book](https://github.com/BlaXpirit/crsfml/tree/tutorials) with latest Gitbook just now and it works well.

**Comment by oprypin** [comment]
New version of Gitbook broke the assumptions made in my plugin. Will fix, hopefully today.

**Comment by nagim** [comment]
Thanks for your time! I tested as well and for me, it ungreys, if empty section is placed elsewhere, but the folders beneath don't show.

Thanks again very much!

**Comment by oprypin** [comment]
I fixed the problem of being unable to click the link. But it seems like your problem may be different?

Anyway, you can try the new version that I pushed and also make sure this isn't some other plugin causing this (or an error in SUMMARY, but I don't see anything wrong).

If you still can't figure this out, maybe give a screenshot of your problem. Publishing the sources to whole book would be helpful but it's OK if that's not possible.

**Comment by nagim** [comment]
Thanks, the update fixed it for me (used ungrey version 1.2.0, GitBook version 2.6.6). Thank you very much for the quick fix. You can close the issue.
